xbuild/src/gradle/settings.gradle

```diff
@@ -14,3 +14,4 @@ dependencyResolutionManagement {
 }
 
 include ':app'
+include ':baseAssets'
```

Comment:
Note that this folder/file is optional, and I'm a tad confused that `gradle` doesn't complain when it doesn't exist.

The `app` would obviously complain when there's `assetPacks = [":baseAssets"]` to a pack that doesn't exist, obviously :)